ietfdata/mailarchive.py

```diff
@@ -137,11 +212,13 @@ def last_updated(self) -> datetime:
 class MailArchive:
```

Comment:
Should this be a `MailArchiveExt` class? I added `DataTrackerExt` for extensions to the datatracker class that perform complex queries and integrate across data sources. This seems like it might be in the same category?

Reply:
Pulled all of the metadata classes and methods into mailarchive_ext.py in the latest commits.